Trellix/trellix-edr/tests/affectedhost_event.json

```
@@ -0,0 +1,42 @@
+{
+  "input": {
+    "sekoiaio": {
+      "intake": {
+        "dialect": "trellix-edr",
+        "dialect_uuid": "954a6488-6394-4385-8427-621541e881d5"
+      }
+    },
+    "message": "{\"id\":\"649889\",\"type\":\"affected-hosts\",\"threatId\":\"182612\",\"attributes\":{\"detectionsCount\":0,\"severity\":\"s0\",\"rank\":270,\"firstDetected\":\"2023-09-25T15:00:20.994Z\",\"host\":{\"aGuid\":\"6D0A37A8-B5B7-4414-9444-A2B17721642B\",\"hostname\":\"hostname\",\"epoTags\":[\"string\"],\"os\":{\"major\":0,\"minor\":0,\"build\":0,\"sp\":\"string\",\"desc\":\"string\"},\"netInterfaces\":[{\"name\":\"string\",\"macAddress\":\"mac1\",\"ip\":\"1.2.3.4\",\"type\":0},{\"name\":\"string\",\"macAddress\":\"mac2\",\"ip\":\"1.2.3.5\",\"type\":0},{\"name\":\"string\",\"macAddress\":\"mac3\",\"type\":0},{\"name\":\"string\",\"ip\":\"1.2.3.6\",\"type\":0}],\"lastBootTime\":\"2023-09-25T15:00:20.994Z\",\"traceExtendedVisibility\":0,\"hostOs\":\"string\"}}}"
+  },
+  "expected": {
+    "message": "{\"id\":\"649889\",\"type\":\"affected-hosts\",\"threatId\":\"182612\",\"attributes\":{\"detectionsCount\":0,\"severity\":\"s0\",\"rank\":270,\"firstDetected\":\"2023-09-25T15:00:20.994Z\",\"host\":{\"aGuid\":\"6D0A37A8-B5B7-4414-9444-A2B17721642B\",\"hostname\":\"hostname\",\"epoTags\":[\"string\"],\"os\":{\"major\":0,\"minor\":0,\"build\":0,\"sp\":\"string\",\"desc\":\"string\"},\"netInterfaces\":[{\"name\":\"string\",\"macAddress\":\"mac1\",\"ip\":\"1.2.3.4\",\"type\":0},{\"name\":\"string\",\"macAddress\":\"mac2\",\"ip\":\"1.2.3.5\",\"type\":0},{\"name\":\"string\",\"macAddress\":\"mac3\",\"type\":0},{\"name\":\"string\",\"ip\":\"1.2.3.6\",\"type\":0}],\"lastBootTime\":\"2023-09-25T15:00:20.994Z\",\"traceExtendedVisibility\":0,\"hostOs\":\"string\"}}}",
+    "event": {
+      "kind": "event",
+      "start": "2023-09-25T15:00:20.994000Z"
+    },
+    "host": {
+      "id": "6D0A37A8-B5B7-4414-9444-A2B17721642B",
+      "ip": ["1.2.3.4", "1.2.3.5", "1.2.3.6"],
+      "mac": ["mac1", "mac2", "mac3"],
+      "name": "hostname",
+      "os": {
+        "full": "string",
+        "version": "0 0 0"
```

I think that a dot `.` is more suitable than a space ` ` to split the different part of the version.
```suggestion
        "version": "0.0.0"
```